src/commands/config.ts

```diff
@@ -10,7 +10,8 @@ import { COMMANDS } from '../CommandsEnum';
 export enum CONFIG_KEYS {
   OPENAI_API_KEY = 'OPENAI_API_KEY',
   description = 'description',
-  emoji = 'emoji'
+  emoji = 'emoji',
+  skipCommitMessageCheck = 'skipCommitMessageCheck'
```

Comment:
lets rename it to `skipCommitMsgHookSourceCheck`?

Reply:
Done!

Reply:
awesome @Lars- thanks! could you change base branch to `dev` instead of `master` pls <3

Reply:
Also done!

Reply:
@Lars- there is a conflict, could you solve pls and let's merge it!

Reply:
The issue wasn't there when I sent the PR, so you may change it